helm/fdb-instance/templates/local_cluster.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps.foundationdb.org/v1beta1
```

Comment:
The local_cluster file is designed for local development, so I don't think we want to use it as part of a helm chart. Most of the things that are defined in this sample are things that aren't as likely to be good fits for real clusters. In general, I'm not sure how much value there is having a helm chart for the CRD, rather than just having the customization go through the CRD.

Reply:
Hi @brownleej 
It totally depends on us, is we want to get the CR,which in our case is a FoundationDBCluster, created as part of helm or not. By in my experience some database use this approach where they install the operator through one helm installation and actual database (CR) through another.
What I suggest is to install the CR using second helm deployment and improve that for real use. WDYT